test/blackbox/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ macro(add_blackbox_gtest)
                 list(GET GTEST_TEST_NAME 1 GTEST_GROUP_NAME)
                 list(GET GTEST_TEST_NAME 3 GTEST_TEST_NAME)
                 add_test(NAME ${test}.${GTEST_GROUP_NAME}.${GTEST_TEST_NAME}
+                    WORKING_DIRECTORY "/data/local/tmp"
```

Comment:
This should only be done for android, and the value should be set to whatever ANDROID_TESTING_ROOT contains

Reply:
If everything works as it should, I think this line can be removed